applications/StructuralMechanicsApplication/custom_response_functions/structural_response.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import print_function, absolute_import, division
+
+# importing the Kratos Library
+from KratosMultiphysics import *
+import structural_mechanics_analysis
+
+import time as timer
+
+class ResponseFunctionBase(object):
+    def __init__(self, identifier, project_parameters):
+        self.identifier = identifier
+        self.project_parameters = project_parameters
+    def Initialize(self):
+        pass
+    def CalculateValue(self):
+        raise NotImplementedError("CalculateValue needs to be implemented by the base class")
+    def CalculateGradient(self):
+        raise NotImplementedError("CalculateGradient needs to be implemented by the base class")
+    def GetShapeGradient(self):
+        raise NotImplementedError("GetShapeGradient needs to be implemented by the base class")
+    def Finalize(self):
+        pass
+
+class SimpleResponseFunctionWrapper(ResponseFunctionBase):
+    def __init__(self, identifier, project_parameters, response_function_utility, model_part = None):
+        super(SimpleResponseFunctionWrapper, self).__init__(identifier, project_parameters)
+
+        # Create the primal solver
```

Comment:
The comments within the classes in this file are unnecessary and prone to become outdated. The code is already descriptive enough.